src/Main.elm

```diff
@@ -632,17 +675,140 @@ modalView : Model -> Html Msg
 modalView model =
     div maskStyle
         [ modalContent []
-            [ windowsChrome [ style "padding" "0 18px 18px" ]
+            [ windowsChrome [ style "padding" "0 18px 90px" ]
                 [ formGroup "Difficulty"
                     [ radiobutton "Beginner" Beginner model.game
                     , radiobutton "Intermediate" Intermediate model.game
                     , radiobutton "Expert" Expert model.game
+                    , customRadioButton model.menu
                     ]
                 ]
             ]
         ]
 
 
+testDisplay : Maybe Menu -> Html Msg
+testDisplay menu =
+    label [ style "display" "flex", style "align-items" "center" ]
+        [ input
+            [ type_ "radio"
+            , name "value"
+            , onClick (OpenMenu (CustomDifficultyMenu 50 50 50))
+            , checked True
+            , style "margin" "4px 8px"
+            ]
+            []
+        , text "Custom Difficulty"
+        ]
+
+
+radiobutton : String -> Difficulty -> Difficulty -> Html Msg
+radiobutton settingLabel difficulty currentGameDifficulty =
+    let
+        bombCount =
+            getBombCount difficulty
+
+        x =
+            first (getDimensions difficulty)
+
+        y =
+            second (getDimensions difficulty)
```

Comment:
you can say

```elm
(x, y) =
  getDimensions difficulty
```

Reply:
I could remove that whole section actually, seeing that those variables aren't being used at all.